lms/djangoapps/support/serializers.py

```diff
@@ -54,10 +61,15 @@ class ProgramEnrollmentSerializer(serializers.Serializer):
     status = serializers.CharField()
     program_uuid = serializers.UUIDField()
     program_course_enrollments = ProgramCourseEnrollmentSerializer(many=True)
+    program_name = serializers.SerializerMethodField()
 
     class Meta(object):
         model = ProgramEnrollment
 
+    def get_program_name(self, obj):
+        program_list = get_programs_by_uuids([obj.program_uuid])
+        return next(iter(program_list), {}).get('title', '')
```

Comment:
I didn't know `next` could take a default argument. Nice!

Consider returning a more informative default, like `"<Program title not found>"`.

Reply:
If you look at the UI, you can see that the program_name is used in a title. Returning blank as the default makes the most sense in the Frontend's point of view at the moment. Obviously, this can change when the API client expectation changes in the future.